data/constants/keycodes/keycodes_0.0.7_pointing_modes.hjson

```diff
@@ -0,0 +1,31 @@
+{
+    "ranges": {
+        "0x5302/0x001F": {
+            "define": "QK_POINTING_MODES_MO"
+        },
+        "0x5322/0x001F": {
+            "define": "QK_POINTING_MODES_TG"
+        },
+        "0x5342/0x0001": {
+            "define": "QK_POINTING_MODES_UTIL"
+        }
+    },
+    "keycodes": {
+        "0x5342": {
+            "group": "pointing_modes_util",
+            "key": "QK_PM_CYCLE_DEVICES",
+            "label": "Cycle pointing mode device",
+            "aliases": [
+                "PM_CYDEV"
+            ]
+        },
+        "0x5343":{
+            "group": "pointing_modes_util",
+            "key": "QK_PM_CYCLE_PRECISION",
+            "label": "Cycle through precision values for active device",
+            "aliases": [
+                "PM_CYCPRE"
+            ]
+        }
+    }
+}
```

Comment:
#24848 added keycodes for community modules in `keycodes_0.0.7.hjson` so these changes need to go into `0.0.8`.